website/views/organization.py

```
@@ -3742,3 +3744,373 @@ def extract_payment_info_from_comments(self, issue, owner, repo_name, issue_numb
+            
+            if not sponsorship_id:
+                logger.error(f"No sponsorship ID in response: {graphql_data}")
+                return JsonResponse({
+                    "success": False,
+                    "error": "Failed to create sponsorship. Please ensure the recipient has GitHub Sponsors enabled."
+                }, status=400)
+            
+            logger.info(f"Successfully created GitHub Sponsors payment: {sponsorship_id}")
+            
+        except requests.exceptions.RequestException as e:
+            logger.exception(f"Network error calling GitHub Sponsors API: {e}")
```

Ruff (0.14.0)

Redundant exception object included in `logging.exception` call

(TRY401)